scripts/common.js

```diff
@@ -256,6 +256,23 @@ export async function loadLazy(doc) {
     header.appendChild(subnav);
   }
 
+  // to add the article search block by checking the metadata
```

Comment:
Put all of this inside of a method please :)

Reply:
done